### DISCUSSION TITLE
commonProps question

### DISCUSSION THREAD

**Discussion by Hillsie** [aws-powertools/powertools-lambda-typescript#1789] [Q&A] [answered]
In the CDK example [powertools-layer ](https://github.com/aws-powertools/powertools-lambda-typescript/blob/75dc5b1e16944416a287628c50ec4a0cf82c3023/examples/cdk/src/example-stack.ts#L56C11-L56C23 ), there is a 12 digit number. What's it represent?

**Comment by dreamorosi** [comment, marked as answer]
Hi @Hillsie thank you for taking the time to create this discussion.

The number is the AWS account ID for the account where the public Lambda Layer is hosted. You can see it referenced in the ARN [1] of the layers in our docs: https://docs.powertools.aws.dev/lambda/typescript/latest/#lambda-layer

[1] [Amazon Resource Names (ARNs)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#)

**Reply by Hillsie** [reply]
Thanks @dreamorosi . ~Probably worth a PR . I'll update the example and doco ... stand by ...~
Breaking things. I'll do it when I've figured some things out.
